src/test/scala/Tests.scala

```diff
@@ -1,7 +1,17 @@
 /**
   * Created by Administrator on 18/07/2017.
   */
+import org.scalatest.FunSuite
+import Main.buildString
 
-class Tests extends  {
+class Tests extends FunSuite {
 
+
```

Comment:
You're testing only the biggest numbers, but what about the millions and so on, this is only covering like 1/7th of the functionality even though you might have the coverage percentage...